index.html

```diff
@@ -5,43 +5,185 @@
-          <li class="top-section-list">facebook</li>
+ <main>
+   <!-- Header Starts Here -->
+   <header>
+    <!-- Top Section -->
+    <div class="top-section">
+      <ul class="top-section-ul">
+        <li class="top-section-list"><img src="Assets\icons8-facebook-18.png" alt="facebook"/></li>
+        <li class="top-section-list"><img src="Assets\icons8-twitter-18.png" alt="twitter"/></li>
+        <li class="top-section-list">English</li>
+        <li class="top-section-list">My Page</li>
+        <li class="top-section-list">Logout</li>
+      </ul>
+    </div>
+
+    <!-- Navbar -->
+    <!--Mobile-->
+    <nav class="navbar-mobile">
+      <div class="menu">
+        <a> <img src="Assets\mobile_menu 1.png" class="menubar" alt="navbar"> </a>
+         </div>
+    </nav>
+
+    <nav class="mobile-menu">
+      <ul class="mobile-menu-list">
+       <img src="Assets\icons8-close-24 (6).png" class="closebtn" alt="close">
+       <li class="menu-list"> <a href="about.html" class="menu-link">About</a></li>
+     <li class="menu-list"> <a href="#about" class="menu-link">Program</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Join</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Sponsor</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">News</a></li>
+    </ul>
+     </nav>
+    
+      <!-- Nav Items -->
+      <nav class="navbar-container">
+        <a href="index.html" class="p-logo"><p class="logo">WIT</p></a>
+        <ul class="navbar-items">
+          <li>
+            <a href="about.html" class="menu-link-desktop">About</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Program</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Join</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Sponsor</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">News</a>
+          </li>
+
+          <li class="list-items">
+            <a href="#">WITMP 2023</a>
+          </li>
         </ul>
-      </div>
+      </nav>
+  </header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="theme-info">
+  <!-- Hero -->
+  <div class="first-section">
+  <div>
+  <h4 class="theme-intro">"Great News!"</h4>
+  <h1 class="theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h1>
+</div>
+
+<div class="intro">
+<h4 class="intro-text">
+Are you a woman trying to navigate the tech space? then this programme is for you. join us in Rivers State for the Women In Tech mentorship program 2023 edition.
+</h4>
+</div>
+
+<div class="venue">
+<h3 class="date-text">
+6th - 7th JULY 2023
+</h3>
+<h5 class="location">
+@Landmark Hotel, Rivers State.
+</h5>
+</div>
+</div>
+
+<div class="second-section">
+  <div class="program">
+    <h4 class="main-program">Main Program</h4>
+    <hr class="one">
```

Comment:
[Optional] I noticed that you didn't apply the hx hierarchy rule for good semantics principles, which should imply that all h1 elements should come before h2, h2 before h3, h3 before h4, h4 before h5, h5 before h6. Please kindly implement this for all your HTML pages as it is a major requirement to follow good HTML semantics but this was omitted by the previous reviewer.

Thank you as you implement this and good job on your project 😄

Reply:
Thank you for your review @usorfaitheloho . I will put this into practice.